types/openui5/sap.ui.mdc.d.ts

```diff
@@ -1,4 +1,148 @@
-// For Library Version: 1.118.0
+// For Library Version: 1.119.0
+
+declare module "sap/ui/mdc/AggregationBaseDelegate" {
+  import Control from "sap/ui/mdc/Control";
+
+  import Control1 from "sap/ui/core/Control";
+
+  /**
+   * @since 1.82.0
+   *
+   * Base delegate implementation for {@link sap.ui.mdc.Control MDC Controls}. This delegate serves as base
+   * implementation for aggregation-based controls, such as:
+   *
+   *
+   * 	 - `sap.ui.mdc.Chart`
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/openui5/sap.ui.mdc.d.ts;
  Line: 15;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.